packages/nodes-base/nodes/EditImage.node.ts

```diff
@@ -11,7 +11,7 @@ import {
 	INodeType,
 	INodeTypeDescription,
 } from 'n8n-workflow';
-import * as gm from 'gm';
+const gm = require('gm').subClass({imageMagick: true});
```

Comment:
Hey @MedAliMarz !
I was having a similar issue when using this node - stream yields an empty buffer.

But then I installed GraphicsMagick on my machine (in my case via `dnf install GraphicsMagick` and it worked. So I think we should revert to the previous import as some operations (like draw square) are not working with this implementation.

Reply:
Hey @krynble, i don't know if it is the right choice to let a node depends on a local binary and not solely on npm modules, if there are no issues raised from the users we can revert to the previous import statement otherwise we can fix the broken operations, what do you think ?

Reply:
Please fix as @krynble mentioned above. We depend on that already since the inception of n8n. Thanks!

Reply:
Thanks for your contribution @janober . I was about to ask you for your feedback as Dali has a point here.

Anyway, we would need to further test if we changed the lib as I believe it's not 100% compatible. The draw rectangle operation did not work perfectly in my test.

Reply:
Ah yes, he def. has a point and we should avoid that as much as possible. But in this case it already is in place and works like that for a very long time. As nobody reported any issues I do not see a reason to change it. In the end would it mean that we have to do a lot of testing now in different environments and potentially break the workflows of people as there are so many different things that could go wrong. Like we would have to test all different combinations of operations in order (as that sadly matters) and that best with all different file formats and so on. Additionally, would we also have to do speed tests as the chance that it would get slower is high. Also high chance that the results differ for some operations or do not work at all. So this operations we would have to fix somehow. So in the end would it be a lot of work, still a high risk of breaking something without really a reason to do that now.

Reply:
That's totally reasonable, reverting to the previous import should be the right action for now.

Reply:
Thanks! But again, your thinking is too 100% correct and very good. Just would not do it right now without a good reason.